src/Features/Core/Portable/ExtractMethod/Enums.cs

```diff
@@ -7,7 +7,6 @@ namespace Microsoft.CodeAnalysis.ExtractMethod;
 internal enum DeclarationBehavior
 {
     None,
-    Delete,
```

Comment:
completely unread.  removed since i'm trying to wrap my head around all these values and what tjhey do.